src/browser/util/url.js

```diff
@@ -87,6 +93,19 @@ define(function() {
       return !this.urlValidate.validity.valid;
     },
 
+    isAboutURL: function(url) {
+      return url.startsWith('about:');
```

Comment:
Ideally should use window.URL() and check the protocol (https://developer.mozilla.org/en-US/docs/Web/API/URLUtils/protocol) since it's case insensitive.